azkaban-common/src/main/java/azkaban/executor/container/FlowStatusChangeEventListener.java

```diff
@@ -31,20 +31,20 @@
 import com.google.inject.Inject;
 import java.util.HashMap;
 import java.util.Map;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.log4j.Logger;
 
 /**
- * This class is the event listener for flow executed in a Kubernetes pod
- * It handles event, gets event metadata, and sends metadata to event reporter plugin
+ * This class reports FLOW_STATUS_CHANGED events in web server, flow statuses include
+ * e.g. READY, DISPATCHING, PREPARING
+ * It handles event, prepares event metadata, and sends metadata to event reporter plugin
  */
-public class PodEventListener implements EventListener<Event> {
+public class FlowStatusChangeEventListener implements EventListener<Event> {
```

Comment:
You don't need to keep this class in container package as it is generic for all types of dispatch methods.

Reply:
moved to azkaban/executor